MAUI/DataGrid/Unbound-Column.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: Unbound Columns in MAUI DataGrid control | Syncfusion
+description: Learn here all about Unbound columns support in Syncfusion MAUI DataGrid (SfDataGrid) control and more about it.
```

Comment:
Learn here all about unbound columns support in Syncfusion MAUI DataGrid (SfDataGrid) control and more about it.